src/commands/admin_commands.py

```diff
@@ -26,36 +30,26 @@ async def check_admin(self, interaction: Interaction) -> bool:
             logging.info(f"User {user_name} is authorized.")
             return True
         else:
+            await interaction.response.send_message("You don't have permission to execute that command.", ephemeral=True)
             logging.warning(f"User {user_name} is not authorized.")
             return False
 
-    async def interaction_check(self, interaction: Interaction) -> bool:
-        """Restrict all admin commands visibility to authorized users only."""
-        is_admin = await self.check_admin(interaction)
-        if is_admin:
-            return True
-        await interaction.response.send_message("Unauthorized", ephemeral=True)
-        return False
-
     @app_commands.command(
-        name="log-info", description="Display all current environment variables."
+        name="log-variables", description="Display all current environment variables."
     )
     async def log_info(self, interaction: Interaction):
         """Command to log and display all relevant environment variables."""
         if not await self.check_admin(interaction):
-            await interaction.response.send_message("Unauthorized", ephemeral=True)
             return
 
-        # Collect environment variable values
-        guild_id = os.getenv("GUILD_ID", "Not Set")
-        skullboard_channel_id = os.getenv("SKULLBOARD_CHANNEL_ID", "Not Set")
-        required_reactions = os.getenv("REQUIRED_REACTIONS", "Not Set")
-        tenor_api_key = os.getenv("TENOR_API_KEY", "Not Set")
-        gemini_api_key = os.getenv("GEMINI_API_KEY", "Not Set")
+        # Get values from database instead of env
+        guild_id = self.settings_db.get_setting("GUILD_ID") or "Not Set"
+        skullboard_channel_id = self.settings_db.get_setting("SKULLBOARD_CHANNEL_ID") or "Not Set"
+        required_reactions = self.settings_db.get_setting("REQUIRED_REACTIONS") or "Not Set"
 
-        # Construct a formatted message for environment variables
+        # Construct message as before
         config_message = (
-            "**Current Environment Variables:**\n"
+            "**Current Settings:**\n"
             f"Guild ID: `{guild_id}`\n"
             f"Skullboard Channel ID: `{skullboard_channel_id}`\n"
             f"Required Reactions: `{required_reactions}`\n"
```

Comment:
Would be nice to use an [embed](https://discordpy.readthedocs.io/en/stable/api.html#embed) for this.